tutorials/cloud-iot-firestore-config/functions/package.json

```diff
@@ -27,7 +27,7 @@
   "devDependencies": {
     "gts": "^0.5.4",
     "tslint": "^5.8.0",
-    "typescript": "~2.6.1"
+    "typescript": "~2.8.3"
```

Comment:
we might as well upgrade to the 3x latest branch and test - it should all still work

Reply:
I tried with 3x version but it had some other issues.